README.md

```diff
@@ -1 +1,144 @@
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 hello-react-front-end <a name=" hello-react-front-end"></a>
+
+   **hello-react-front-end** is a simple react app that uses Raila API to display greetings in different languages
+   
+   Checkout the [Rails-API](https://github.com/torobucci/hello-rails-back-end) used.
+  
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<ul>
+    <li><a href="https://www.w3schools.com/js/">Javascript</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/html/">HTML5</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Messages component**
+- **Display random messages**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to install Node:
+
+```sh
+ Node.js
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone git@github.com/torobucci/hello-react-front-end.git
+  cd hello-react-front-end
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run start
+```
+
+<!-- AUTHORS -->
+
+## 👥 Author : <a name="authors">Kevin Toro</a>
+- GitHub: [@torobucii](https://github.com/torobucci)
+- Twitter: [@torobucii](https://twitter.com/@torobucii)
+- LinkedIn: [Kevin Toro](https://www.linkedin.com/in/kevin-toro-047181238/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Authentication Page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give me a star⭐
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [Microverse](https://github.com/microverseinc) for providing the template for this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/torobucci/Portfolio-finish-mobile-version/blob/main/MIT-LICENSE.txt) licensed.
```

Comment:
- [ ] 🚧 You did a great job documenting your project! 💯 However please make sure that you have a LICENSE and that its respective link in the README is leading to it. Currently, it is pointing to your portfolio's license. You can follow these steps from [GitHub's official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to set up a LICENSE. Microverse suggests using MIT! 🤓 🔧